lepton/image.go

```diff
@@ -62,11 +62,11 @@ func addDNSConfig(m *fs.Manifest, c *types.Config) {
 	}
 	err := os.WriteFile(resolv, data, 0644)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed save dns in temporary resolv.conf, error: %s", err)
 	}
 	err = m.AddFile("/etc/resolv.conf", resolv)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed add resolv.conf, error: %s", err)
 	}
```

Comment:
since this was a panic need to exit here

Reply:
done